src/style/index.scss

```diff
@@ -198,9 +198,147 @@ $landing-color-dark-bg: $color-hack-black-5;
     max-width: 100%;
     height: auto;
   }
+  /*--  FAQ-SECTION  --*/
+
+  /* section.FAQ{//OPTION 1 FOR FAQ BACKGROUND (COMMENT THIS OUT WHEN LOOKING AT OPTION 2)
+    background-image: none;
+    @include media-query($md-up) {
+      background-image: url("/static/faq-symbols.svg");
+      background-size: stretch;
+    }
+  }*/
+
+   section.FAQ .container {
+    margin: 6 auto;
+    padding-top: 50rem;
+    padding-bottom: 18rem;
+    width: 85rem;
+    text-align: center;
+  }
+  
+  .flex-container{
+    display: flex;
+    justify-content: space-between;
+  }
+  .left-container{
+    width: 40rem;
+    text-align: initial;
+  }
+
+  .right-container{
+    width: 40rem;
+    text-align: initial;
+  }
+
+  .accordion a {
+    position: relative;
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -webkit-flex-direction: column;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    width: 100%;
+    padding: 1rem .5rem 1rem 1rem;
+    color: $color-hack-black-80;    
+  }
+  
+  .background{
+    background-color:$color-hack-black-5;
+    box-shadow: 0px 2px 2px #E9E9E9;
+    border-radius: 4px;
+  }
+
+  .accordion a:hover,
+  .accordion a:hover::after {
+    cursor: pointer;
+    color: $color-hack-red-med;
+  }
+  
+  .accordion a.active {
```

Comment:
Should combine all the states and stuff into one big block using the `&` SCSS variable.

```css
.accordion__link {
     // some styling
    &:hover {
        // hover styling
    }
}
```